common/arch/win32/rbaudio.cpp

```diff
@@ -0,0 +1,530 @@
+/*
```

Comment:
This file looks very familiar.  Why are you copying it instead of patching the existing file?  What are your plans for avoiding duplicate work maintaining separate copies of the file, one for SDL1 and one for SDL2, in different paths?

Reply:
common/arch/sdl/rbaudio.cpp is for SDL1 which has its own CD code, common/arch/win32/rbaudio.cpp will be for SDL2 which does _not_ have its own CD code (it was removed in SDL2 and there is no replacement, as their documentation says), so instead it uses a Windows-exclusive API to play from the CD directly without using SDL as the middle man.

Reply:
I am aware SDL removed their support.  In my initial review, it looked like there would be a substantial amount of code common between the two implementations.  In reviewing a `diff` between the two files, I see that even after suppressing whitespace noise, there is very little in common.  I withdraw my objection here.